superset/connectors/sqla/models.py

```diff
@@ -841,6 +858,7 @@ def get_sqla_query(  # sqla
 
         where_clause_and = []
         having_clause_and: List = []
+        assert filter
```

Comment:
Note this could be `None` per the function signature and thus line #862 would throw an exception, however to the best of my knowledge this hasn't happened and thus `filter` is probably _never_ `None`.